.travis.yml

```diff
@@ -70,8 +72,25 @@ matrix:
       python: "3.6"
       env: TOXENV=py36-flake8
 
+    - stage: Deploy
+      name: Publishing current Git tagged version of dist to PyPI
+      if: tag IS present
+      install: skip
+      script: skip
+      deploy:
+        provider: pypi
+        user: any-maintainer-on-pypi  # I recommend having a separate account
+        password:
+          secure: >  # needs real data
+            PLACE_FOR_AN_ENCRYPTED_STRING
```

Comment:
@gundalow this will also need to be filled in.

Reply:
I hated that I couldn't set this value easily: I literally had to download the travis gem and use it to encrypt the value. Obviously it also implies that you are sharing your PyPI password with travis.

Reply:
@TomasTomecek Do you know if there is a better way of doing this?

Reply:
Other ways are harder to maintain: more infra management on our side. As for the password, you can add it as a `PYPI_PASSWORD` secret var in Travis CI UI if you don't like embedding it into the config, but personally, I prefer using config.

Reply:
We developed a bot which does this for us. The down side is as @webknjaz says: you need to run it somewhere on your infra. https://github.com/user-cont/release-bot

Reply:
Also:
1) and doesn't match the workflow
2) will probably require adjustments
3) config is separate so that it's unobvious to follow just looking at the CI pipelines one
4) is not reusable (I mean you cannot just hit install on GitHub, bound to actual user account)

Reply:
P.S. I recognize that Travis CI has some flaws. However, my experience is probably also different from yours. I've been contributing to their deployment tool, for example.

Reply:
I wasn't implying you should use our release-bot. It's definitely easier for you to use travis. I just wanted to share a story how we tackled the issue of automating releases.